ui/pages/Project/components/RnaSeqOutliersTable.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Segment, Header } from 'semantic-ui-react'
+
+import { camelcaseToTitlecase } from 'shared/utils/stringUtils'
+import { GeneSearchLink } from 'shared/components/buttons/SearchResultsLink'
+import DataTable from 'shared/components/table/DataTable'
+import FamilyReads from 'shared/components/panel/family/FamilyReads'
+import { COVERAGE_TYPE, JUNCTION_TYPE } from 'shared/components/panel/family/constants'
+import { ButtonLink } from 'shared/components/StyledComponents'
+
+const ScrollableSegment = styled(Segment)`
+  overflow: auto;
+  max-height: 400px;
+`
+
+const RNA_SEQ_SPLICE_NUM_FIELDS = ['zScore', 'pValue', 'deltaPsi']
+const RNA_SEQ_SPLICE_LOC_FIELDS = ['chrom', 'start', 'end', 'strand']
+const RNA_SEQ_SPLICE_DETAIL_FIELDS = ['type', 'readCount', 'rareDiseaseSamplesWithJunction', 'rareDiseaseSamplesTotal']
```

Comment:
@bw2 'type', rareDiseaseSamplesWithJunction', and 'rareDiseaseSamplesTotal' are listed in the original ticket as being in the input data but it does not say that they should be shown in the table. Do you think this data needs to be surfaced in this table, and if so do you think this is the best way to show it?